### PR TITLE
ci: add cla workflow

### PR DESCRIPTION
**Pull request by kentakozuka**

### CODE REVIEW

File: .github/workflows/cla.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        if: (github.event.comment.body == 'recheck' || github.event.comment.body == '/cla sign') || github.event_name == 'pull_request_target'
+        uses: contributor-assistant/github-action@a895a435fcce79ecf28fbce61a4ef0f0dabc9853 # v2.3.1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          PERSONAL_ACCESS_TOKEN: ${{ secrets.CLA_PAT }}
+        with:
+          remote-organization-name: bucketeer-io
+          remote-repository-name: bucketeer-cla
+          path-to-signatures: "cla/signatures.json"
+          custom-notsigned-prcomment: "Welcome!\nThanks for your contribution. It looks like this is your first PR to this repository 🎉. Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA). Please read the [CLA](https://github.com/bucketeer-io/bucketeer/blob/main/CLA.md) and leave a below comment on this pull request to sign the CLA."
```
Comment (cre8ivejp) [inline .github/workflows/cla.yaml:22]: Please add the CLA.md file to this repository instead of the core one.

File: .github/workflows/cla.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+jobs:
+  sign:
+    runs-on: ubuntu-latest
+    steps:
+      - name: "CLA Assistant"
+        if: (github.event.comment.body == 'recheck' || github.event.comment.body == '/cla sign') || github.event_name == 'pull_request_target'
+        uses: contributor-assistant/github-action@a895a435fcce79ecf28fbce61a4ef0f0dabc9853 # v2.3.1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          PERSONAL_ACCESS_TOKEN: ${{ secrets.CLA_PAT }}
```
Comment (cre8ivejp) [inline .github/workflows/cla.yaml:17]: Please set the `CLA_PAT` secret.

File: .github/workflows/cla.yaml (hunk 1 above)
Comment (kentakozuka) [reply .github/workflows/cla.yaml:22]: Can you tell me why we need another CLA file for this repository? I think it is easy to maintain to have only one CLA file.
Do you want to have independent CLA file and signatures.json for each repository?

Comment (cre8ivejp) [reply .github/workflows/cla.yaml:22]: > Do you want to have independent CLA file and signatures.json for each repository?

I understand that the `signatures.json` supports multiple repositories.
The reason is that this repository has no reference to the CLA terms. So how can you sign something that you can't check the terms?

Another reason that I didn't add the CLA to the SDK repositories is that we might not need it.
PipeCD has removed it. You can check the conversation [here](https://github.com/cncf/sandbox/issues/12#issuecomment-1540247323).

Comment (kentakozuka) [reply .github/workflows/cla.yaml:22]: I got it. Groups cannot be an legal entity🤔  Then, let's remove CLA from bucketeer-io org 👍

**Comment by kentakozuka** [comment]
CLA is not necessary. See [here](https://github.com/bucketeer-io/node-server-sdk/pull/23#discussion_r1483779102).

**Pull request closed without merging**